docs/src/Home.js

```diff
@@ -38,7 +38,7 @@ var Home = React.createClass({
         <DocsHeader float={true} />
 
         <HomeSection texture={'url(img/home_intro.png)'} primary={true}>
-          <GrommetLogo large={true} />
+          <GrommetLogo large={true} a11yTitle=""/>
```

Comment:
`a11yTitle` is empty string because the Grommet logo in this context is decoration. The word Grommet is right below it (see line 42).  If `a11yTitle` was left out a screen reader would say the word grommet twice: once for the image and then the again for line 42.